webrender/src/renderer.rs

```diff
@@ -1425,34 +1429,27 @@ impl Renderer {
                 &PrimitiveBatchData::Blend(ref ubo_data) => {
                     self.gpu_profile.add_marker(GPU_TAG_PRIM_BLEND);
                     let shader = self.ps_blend.get(&mut self.device);
-                    self.device.bind_program(shader, &projection);
-                    self.device.bind_vao(self.quad_vao_id);
-
-                    for chunk in ubo_data.chunks(self.max_prim_blends) {
-                        let ubo = self.device.create_ubo(&chunk, UBO_BIND_DATA);
-
-                        self.device.draw_indexed_triangles_instanced_u16(6, chunk.len() as i32);
-                        self.profile_counters.vertices.add(6 * chunk.len());
-                        self.profile_counters.draw_calls.inc();
-
-                        self.device.delete_buffer(ubo);
-                    }
+                    let max_prim_items = self.max_prim_blends;
+                    self.draw_ubo_batch(ubo_data, shader,
```

Comment:
interesting. @glennw do you remember why `draw_ubo_batch` wasn't used here in the first place?